src/Examples/Flowsheets/HDA_flowsheet_with_costing.ipynb

```diff
@@ -0,0 +1,1598 @@
+      "  42r 0.0000000e+00 2.20e+01 6.08e+03  -4.3 5.03e+00    -  1.00e+00 1.25e-01h  4\n",
+      "  43r 0.0000000e+00 8.71e+00 1.09e+03  -4.3 1.86e+00    -  1.00e+00 1.00e+00h  1\n",
+      "  44r 0.0000000e+00 7.26e-01 2.92e-01  -4.3 7.34e-01    -  1.00e+00 1.00e+00h  1\n",
+      "  45r 0.0000000e+00 7.42e-01 1.77e-01  -4.3 1.24e-02    -  1.00e+00 1.00e+00h  1\n",
+      "  46r 0.0000000e+00 7.42e-01 8.16e-07  -4.3 2.28e-05    -  1.00e+00 1.00e+00h  1\n",
+      "  47r 0.0000000e+00 1.14e+01 6.85e+03  -6.4 5.47e+00    -  1.00e+00 6.84e-01f  1\n",
+      "  48r 0.0000000e+00 2.37e+00 1.34e+03  -6.4 2.29e+00    -  9.42e-01 7.43e-01f  1\n",
+      "  49r 0.0000000e+00 5.25e-01 3.05e+02  -6.4 2.31e+00    -  8.86e-01 5.92e-01f  1\n",
+      "iter    objective    inf_pr   inf_du lg(mu)  ||d||  lg(rg) alpha_du alpha_pr  ls\n",
+      "  50r 0.0000000e+00 3.85e-01 1.12e+02  -6.4 4.84e-01    -  1.00e+00 7.58e-01f  1\n",
+      "  51r 0.0000000e+00 1.05e-01 5.46e+01  -6.4 2.35e-01    -  1.00e+00 1.00e+00f  1\n",
+      "  52  0.0000000e+00 5.48e-01 4.23e+06  -1.0 5.08e+00    -  9.90e-01 1.00e+00f  1\n",
+      "  53  0.0000000e+00 3.73e-08 1.05e+03  -1.0 5.37e-01    -  9.90e-01 1.00e+00h  1\n",
+      "Cannot recompute multipliers for feasibility problem.  Error in eq_mult_calculator\n",
+      "\n",
+      "Number of Iterations....: 53\n",
+      "\n",
+      "                                   (scaled)                 (unscaled)\n",
+      "Objective...............:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Dual infeasibility......:   1.0000038222099323e+07    1.0000038222099323e+07\n",
+      "Constraint violation....:   2.9103830456733704e-11    3.7252902984619141e-08\n",
+      "Complementarity.........:   0.0000000000000000e+00    0.0000000000000000e+00\n",
+      "Overall NLP error.......:   2.9103830456733704e-11    1.0000038222099323e+07\n",
+      "\n",
+      "\n",
+      "Number of objective function evaluations             = 64\n",
+      "Number of objective gradient evaluations             = 13\n",
+      "Number of equality constraint evaluations            = 64\n",
+      "Number of inequality constraint evaluations          = 0\n",
+      "Number of equality constraint Jacobian evaluations   = 55\n",
+      "Number of inequality constraint Jacobian evaluations = 0\n",
+      "Number of Lagrangian Hessian evaluations             = 53\n",
+      "Total CPU secs in IPOPT (w/o function evaluations)   =      0.108\n",
+      "Total CPU secs in NLP function evaluations           =      0.003\n",
+      "\n",
+      "EXIT: Optimal Solution Found.\n"
+     ]
+    }
+   ],
+   "source": [
+    "results_m = solver.solve(m, tee=True, symbolic_solver_labels=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Check solver solve status and physical units consistency for model m\n",
```

Comment:
Drop the redundant solve here  (solver solve status).

Unit consistency checks should always happen before you start solving anything.